src/CMakeLists.txt

```diff
@@ -407,6 +421,16 @@ if(USE_VARIABLE_SPLASH_SCREEN)
   target_compile_definitions(mudlet PRIVATE INCLUDE_VARIABLE_SPLASH_SCREEN)
 endif()
 
+if(WIN32)
+  if(USE_MAIN_BUILD_SYSTEM)
+    # The UNICODE and _UNICODE definitions are needed to make
+    # (QString) mudlet::getShortPathName(const QString& name) work:
+    target_compile_definitions(mudlet PRIVATE INCLUDE_MAIN_BUILD_SYSTEM UNICODE _UNICODE)
+  else()
+    target_compile_definitions(mudlet PRIVATE UNICODE _UNICODE)
```

Comment:
This can be outside the `MAIN_BUILD_SYSTEM`-if. Repeated calls to the function append compile definitions.

Reply:
Will do...

Reply:
... done.